packages/minimal/src/managers/resource-manager-dispensers.ts

```diff
@@ -0,0 +1,63 @@
+import { RESOURCE_PREFIX } from './resource-manager';
```

Comment:
Duplicated code, at first glance, this probably better belongs in the new util/tools package, but the problem is that this is runtime code that depends on build time configurations, I expect the util/tools package to contain mostly build time tools, so including this in it would force dependants to add a build tools package as a runtime dependency.

So maybe we should move this to `react-components` instead, this does create a hidden dependency on the tools package for `react-components`, but maybe thats not a problem.

Another option (which on 2nd thought, probably doesn't work) is to simplify the resource loading, icons could be loaded via a standard path so we don't need extra logic of parsing a descriptor files. But there are also things like dispenser locations, which should have never been part of the configuration in the first place, but alas rmf lacks the api for us to obtain those info so we have to embed it. We can move these info into the api server, but that doesn't really solve the problem, now the api server needs to embed these info instead.